lib/api/core/plugins/pluginsManager.js

```diff
@@ -312,34 +345,124 @@ class PluginsManager {
 }
 
 /**
- * Start the plugin worker with its configuration with PM2 when the connection is done
+ * Loads installed plugins in memory
  *
- * @param {object} plugin
- * @param {string} pluginName
- * @param {object} kuzzleConfig
+ * @param {object} config - plugins configuration
+ * @param {number} kuzzleVersion
+ * @param {string} rootPath - Kuzzle root directory
+ * @returns {object} list of loaded plugin
  */
-function initWorkers (plugin, pluginName, kuzzleConfig) {
-  return pm2Promise
-    .then(() => {
-      const pm2StartPromise = Bluebird.promisify(pm2.start, {context: pm2});
+function loadPlugins(config, kuzzleVersion, rootPath) {
+  const pluginsDir = path.resolve(path.join(rootPath, 'plugins/enabled'));
+  const loadedPlugins = {};
 
-      return pm2StartPromise({
-        name: kuzzleConfig.plugins.common.workerPrefix + pluginName,
-        script: path.join(__dirname, 'pluginsWorkerWrapper.js'),
-        execMode: 'cluster',
-        instances: plugin.config.threads,
-        killTimeout: plugin.config.killTimeout || 6000,
-        maxMemoryRestart: plugin.config.maxMemoryRestart || '1G',
-        watch: false
-      });
-    })
-    .catch(err => {
-      if (err) {
-        return Bluebird.reject(new PluginImplementationError('Error while starting worker plugin: '.concat(pluginName)));
+  const pluginList = fs.readdirSync(pluginsDir)
+    .filter(element => {
+      const pluginPath = path.join(pluginsDir, element);
+
+      try {
+        const elStat = fs.statSync(pluginPath);
+        return elStat.isDirectory();
+      }
+      catch(e) {
+        throw new PluginImplementationError(`Unable to load plugin from path "${pluginPath}"; this may occurs with dead symbolic links; ${e.message}`); // eslint-disable-line no-console
```

Comment:
either `this may occur` (without a trailing 's' at the end of 'occur') or, better yet, just remove this part: the error message from `fs.statSync` should be self-explanatory